properties/P000133.md

```diff
@@ -0,0 +1,21 @@
+---
+uid: P000133
+slug: linearly_orderable
```

Comment:
Question for myself: Slugs are everywhere.  But what is a slug, what does it do?